rs/ic_os/config/src/lib.rs

```diff
@@ -79,7 +79,7 @@ mod tests {
         };
         let setupos_settings = SetupOSSettings;
         let hostos_settings = HostOSSettings {
-            vm_memory: 490,
+            vm_memory: 42,
```

Comment:
I tried to remove references to `vm_memory`, and `490` wherever possible, just so it doesn't trip us up fuzzy searching.